libs/wingsdk/src/simulator/simulator.ts

```diff
@@ -931,7 +931,7 @@ export class Simulator {
               status: "failure",
             },
             type: TraceType.RESOURCE,
-            level: LogLevel.VERBOSE,
+            level: LogLevel.ERROR,
```

Comment:
Not sure what to do here, but I think we need to somehow distinguish caught error and uncaught errors